tests/compose/test-basic.sh

```diff
@@ -61,9 +61,6 @@ assert_file_has_content_literal autovar.txt 'd /var/tmp 1777 root root - -'
 assert_file_has_content_literal autovar.txt 'd /var/lib/polkit-1 0750 root polkitd - -'
 echo "ok autovar"
 
-# Validate this exists
-ostree --repo="${repo}" ls "${treeref}" /usr/lib/systemd/system/multi-user.target.wants/ostree-remount.service
```

Comment:
Just noting it explicitly: this filepath is now created at runtime by the ostree generator patch linked above.
So while this static content check can't detect it (and needs to be dropped), on a booted system the filepath will eventually be present.
The ostree CI already tests this.